public/assets/sass/style.scss

```diff
@@ -297,6 +352,19 @@ kb-file-tree {
     display: none;
   }
 }
+
+.status-bar-selector {
+  bottom: 30px;
+  top: initial !important;
```

Comment:
this div is getting injected by angular-material directly to the `<body>`, how can I not use the important by making the selector more specific?